lib/plugins/aws/invokeLocal/fixture/asyncHandlerWithSuccess.js

```diff
@@ -1,51 +1,50 @@
 'use strict';
 
-module.exports.withError = () => Promise.reject(new Error('failed'));
+module.exports.withError = () => new Error('failed');
 
-module.exports.withMessage = () => Promise.resolve('Succeed');
+module.exports.withMessage = () => 'Succeed';
 
 module.exports.withErrorByDone = (event, context) => {
   context.done(new Error('failed'));
 
   // this should not actually trigger a success
-  return Promise.resolve('this should not trigger');
+  return 'this should not trigger';
 };
 
 module.exports.withMessageByDone = (event, context) => {
   context.done(null, 'Succeed');
-  return Promise.resolve('this should not trigger');
+  return 'this should not trigger';
 };
 
 module.exports.withMessageByCallback = (event, context, callback) => {
   callback(null, 'Succeed');
 
-  return Promise.resolve();
+  return;
 };
 
-module.exports.withMessageAndDelayByCallback = (event, context, callback) => {
-  setTimeout(() => callback(null, 'Succeed'), 1);
+module.exports.withMessageAndDelayByCallback = async (event, context, callback) => {
```

Comment:
Is wrapping with a Promise necessary here?

Reply:
I didn't think it was, but if you tag it as `async`, you can't then `await` it unless it returns the promise. As it was defined before, you could `await` the timeout. I just thought that tagging it as `async` and returning a promise was more explicit than not tagging it as an asynchronous function. 

If you'd like, I can revert it to its previous form.

Reply:
I feel like we should leave that function in it's previous form